platform/default/mbgl/storage/offline_database.cpp

```diff
@@ -793,17 +793,42 @@ bool OfflineDatabase::evict(uint64_t neededFreeSize) {
     // size, and because pages can get fragmented on the database.
     while (usedSize() + neededFreeSize + pageSize > maximumCacheSize) {
         // clang-format off
+        Statement accessedStmt = getStatement(
+            "SELECT max(accessed) "
+            "FROM ( "
+            "    SELECT accessed "
```

Comment:
Are you sure the inner query is parsed as suggested by the indentation, and not "(select from resources [unordered and unlimited]) UNION ALL (select from tiles ... ordery by ... limit 50)"?